RecoParticleFlow/PFProducer/src/PFEGammaAlgo.cc

```diff
@@ -2070,188 +2022,189 @@ fillPFCandidates(const pfEGHelpers::HeavyObjectCache* hoc,
+      float hOverHe             = eneHcalGsf/(eneHcalGsf + firstEcalGsfEnergy);
+
+      xtra.setDeltaEta(dEtaGsfEcalClust);
       xtra.setSigmaEtaEta(sigmaEtaEta);      
-      
-      HOverHE = Ene_hcalgsf/(Ene_hcalgsf + FirstEcalGsfEnergy);
-      HOverPin = Ene_hcalgsf / Ein_gsf;
-      xtra.setHadEnergy(Ene_hcalgsf);
+      xtra.setHadEnergy(eneHcalGsf);
 
       // Apply bounds to variables and calculate MVA
-      DPtOverPt_gsf = std::max(DPtOverPt_gsf,-0.2f);
-      DPtOverPt_gsf =  std::min(DPtOverPt_gsf,1.0f);  
-      dPtOverPt_gsf = std::min(dPtOverPt_gsf,0.3f);  
-      chi2_gsf = std::min(chi2_gsf,10.0f);  
-      DPtOverPt_kf = std::max(DPtOverPt_kf,-0.2f);
-      DPtOverPt_kf = std::min(DPtOverPt_kf,1.0f);  
-      chi2_kf = std::min(chi2_kf,10.0f);  
-      EtotPinMode = std::max(EtotPinMode,0.0f);
-      EtotPinMode = std::min(EtotPinMode,5.0f);  
-      EGsfPoutMode = std::max(EGsfPoutMode,0.0f);
-      EGsfPoutMode = std::min(EGsfPoutMode,5.0f);  
-      EtotBremPinPoutMode = std::max(EtotBremPinPoutMode,0.0f);
-      EtotBremPinPoutMode = std::min(EtotBremPinPoutMode,5.0f);  
-      DEtaGsfEcalClust = std::min(DEtaGsfEcalClust,0.1f);  
-      SigmaEtaEta = std::max(SigmaEtaEta,-14.0f);  
-      HOverPin = std::max(HOverPin,0.0f);
-      HOverPin = std::min(HOverPin,5.0f);
-      /*
-      std::cout << " **** PFEG BDT observables ****" << endl;
-      std::cout << " < Normalization > " << endl;
-      std::cout << " Pt_gsf " << Pt_gsf << " Pin " << Ein_gsf  
-		<< " Pout " << Eout_gsf << " Eta_gsf " << Eta_gsf << endl;
-      std::cout << " < PureTracking > " << endl;
-      std::cout << " dPtOverPt_gsf " << dPtOverPt_gsf 
-		<< " DPtOverPt_gsf " << DPtOverPt_gsf
-		<< " chi2_gsf " << chi2_gsf
-		<< " nhit_gsf " << nhit_gsf
-		<< " DPtOverPt_kf " << DPtOverPt_kf
-		<< " chi2_kf " << chi2_kf 
-		<< " nhit_kf " << nhit_kf <<  endl;
-      std::cout << " < track-ecal-hcal-ps " << endl;
-      std::cout << " EtotPinMode " << EtotPinMode 
-		<< " EGsfPoutMode " << EGsfPoutMode
-		<< " EtotBremPinPoutMode " << EtotBremPinPoutMode
-		<< " DEtaGsfEcalClust " << DEtaGsfEcalClust 
-		<< " SigmaEtaEta " << SigmaEtaEta
-		<< " HOverHE " << HOverHE << " Hcal energy " << Ene_hcalgsf
-		<< " HOverPin " << HOverPin 
-		<< " lateBrem " << lateBrem
-		<< " firstBrem " << firstBrem << endl;
-      */
-      
-      float vars[] = { lnPt_gsf, Eta_gsf, dPtOverPt_gsf, DPtOverPt_gsf, chi2_gsf,
-                       nhit_kf, chi2_kf, EtotPinMode, EGsfPoutMode, EtotBremPinPoutMode,
-                       DEtaGsfEcalClust, SigmaEtaEta, HOverHE, lateBrem, firstBrem };
+      dPtOverPtGsf        = std::max(dPtOverPtGsf,-0.2f);
+      dPtOverPtGsf        = std::min(dPtOverPtGsf,1.0f);  
+      ptModeErrOverPtGsf  = std::min(ptModeErrOverPtGsf,0.3f);  
+      chi2Gsf             = std::min(chi2Gsf,10.0f);  
+      //dPtOverPtKf         = std::max(dPtOverPtKf,-0.2f);
+      //dPtOverPtKf         = std::min(dPtOverPtKf,1.0f);  
```

Comment:
if the two above are not used anymore but are still necessary for some potential debugging, please add a note, otherwise remove.